src/Shape.cpp

```diff
@@ -205,7 +289,10 @@ bool Shape::intersect(const Pose3D& pose, const Shape& other) const {
             double t_2l = std::min(t_21,t_22);
 
             if (t_1h > t_2l && t_1l < t_2h)
+            {
+                //std::cout << "Intersection DETECTED"<< std::endl;
```

Comment:
Commented code, if you keep the brackets, opening bracket on the same line after a space.